src/model/bpmn/internal/shape/kinds.ts

```diff
@@ -129,17 +129,11 @@ export enum ShapeBpmnMarkerKind {
 }
 
 /**
- * Base name of the BPMN specification for sub-process kinds.
```

Comment:
**thought:** this is not part of the BPMN specification so do not say it in the JSDoc 😸